lib/util/schema-utils.ts

```diff
@@ -91,26 +91,20 @@ export function looseValue<T, U extends z.ZodTypeDef, V>(
   return schemaWithFallback;
 }
 
-export function parseJson<
-  T = unknown,
-  Schema extends z.ZodType<T> = z.ZodType<T>
->(input: string, schema: Schema): z.infer<Schema> {
-  const parsed = JSON.parse(input);
-  return schema.parse(parsed);
-}
+type JsonData =
```

Comment:
[`type-fest`][1] has utility types for JSON objects that might be easier to use than maintaining our own.

[1]: https://github.com/sindresorhus/type-fest#json